packages/tradershub/src/flows/FirstSignup/CitizenshipModal/CitizenshipModal.tsx

```diff
@@ -0,0 +1,87 @@
+    const isBrazil = values.country === 'br';
+
+    useEffect(() => {
+        if (residenceList?.length && clientCountry && values.country === '') {
+            setFieldValue('country', clientCountry);
+        }
+    }, [clientCountry, setFieldValue, residenceList, values.country]);
+
+    // Add <Loading /> here later when it's created
+    if (clientCountryLoading && residenceListLoading) return null;
+
+    return (
+        <Modal className='h-full rounded-default max-w-[328px] md:max-w-[440px]'>
+            <div className='flex flex-col max-w-[296x] md:max-w-[392px] p-16 space-y-16 md:space-y-24 md:p-24'>
+                <Text weight={'bold'}>Select your country and citizenship:</Text>
+                <Dropdown
+                    dropdownIcon={<LabelPairedChevronDownMdRegularIcon />}
+                    errorMessage='Country of residence is where you currently live.'
+                    label='Country of residence'
+                    list={residenceList}
+                    name='country'
+                    onSelect={selectedItem => {
+                        setFieldValue('country', selectedItem);
+                    }}
+                    value={values.country}
+                    variant='comboBox'
+                />
+                <Dropdown
+                    dropdownIcon={<LabelPairedChevronDownMdRegularIcon />}
+                    errorMessage='Select your citizenship/nationality as it appears on your passport or other government-issued ID.'
+                    label='Citizenship'
+                    list={residenceList}
+                    name='citizenship'
+                    onSelect={selectedItem => {
+                        setFieldValue('citizenship', selectedItem);
+                    }}
+                    value={values.citizenship}
+                    variant='comboBox'
+                />
+                {isBrazil && (
+                    <Checkbox
+                        checked={isCheckBoxChecked}
+                        label={
+                            <Text size='sm'>
+                                I hereby confirm that my request for opening an account with Deriv to trade OTC products
+                                issued and offered exclusively outside Brazil was initiated by me. I fully understand
+                                that Deriv is not regulated by CVM and by approaching Deriv I intend to set up a
+                                relation with a foreign company.
+                            </Text>
+                        }
+                        labelClassName='flex-1'
+                        onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                            setIsCheckBoxChecked(event.target.checked)
+                        }
+                        wrapperClassName='w-auto'
+                    />
+                )}
+                <Button
+                    className='w-full md:self-end md:w-fit'
```

Comment:
Do we need the md ?

Reply:
@thisyahlen-deriv agree, need to use `lg` instead